### PR TITLE
I am the Library of Ruina (*turns people into books*)

### PR DESCRIPTION
**Pull request by scionalu**
## About The Pull Request

As the title says, adds a proc that turns people (including all living mobs) into a book
If you have ego gear or if you turned abnormalities into a book, you can get ego loot from burning the book (chosen randomly from your equipment / that abno's pool)

Also adds a cargo item that lets you book an abnormality for free ego, but only if you kill them (costs 1500 PE)
You'll basically get 1 after dusk on lowpop or after noon in mid-highpop
You deserve this anyways if you manage to kill aleph abno with waw gear
Also speeds up farming on lategame with how fast the game is getting

Ah lastly adds a new smite (book) to admemes I guess

You can customize the name, info, and loot drops from the proc if you want, so it's quite convenient for future use

Tldr, how to use custom items for loot instead:
var/list/custom = list(
		/obj/item/toy/plush/melt,
		/obj/item/toy/plush/fumo,
		/obj/item/ego_weapon/twilight,
		/obj/item/gun/ego_gun/flammenwerfer,
		/obj/item/clothing/suit/armor/ego_gear/paradise
	)

user.turn_book(custom)

while the complete var order is: 
(custom loot, item name, page title, page info, and custom icon)

Edit: Forgot to say my sprites are shite, if anyone would update them in the future, that'd be nice. It's decent enough for now

## Why It's Good For The Game

I am the library of ruina

## Changelog
:cl:
add: Added new proc, added new cargo item
soundadd: added book death and burn sfx from lor
imageadd: sprites for general invitation, LoR book, and turning into book anim
admin: gives a new smite
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
One step closer to LOR13

**Review by Kitsunemitsu — APPROVED** [review]
Code looks fine; You will be notified if there are any issues.
